lib/facebook/messenger/welcome.rb

```diff
@@ -0,0 +1,61 @@
+require 'httparty'
+
+module Facebook
+  module Messenger
+    # This module handles setting welcome messages.
+    #
+    # https://developers.facebook.com/docs/messenger-platform/send-api-reference#welcome_message_configuration
+    module Welcome
+      include HTTParty
+
+      base_uri 'https://graph.facebook.com/v2.6/me'
```

Comment:
I think the API endpoint for welcome message is `https://graph.facebook.com/v2.6/<PAGE_ID>/thread_settings?access_token=<PAGE_ACCESS_TOKEN>`

https://developers.facebook.com/docs/messenger-platform/send-api-reference#welcome_message_configuration